client/components/data/query-users/index.jsx

```diff
@@ -37,12 +39,20 @@ class QueryUsers extends Component {
 	}
 
 	request() {
-		this.props.requestUsers( this.props.siteId, this.props.userIds );
+		const { siteId, userIds, currentUserId } = this.props;
+		// Only request users if a list of user Ids is provided. Otherwise, all users would be requested incl. the current user.
+		if ( userIds.length ) {
```

Comment:
Hrm -- question, should we put this behind a prop?

That way we can be certain that our usage is the only one affected.

Reply:
As discussed in Slack -- Post revisions is literally the only place in Calypso where `QueryUsers` is currently used. We just need to describe that the current user is specifically filtered out in the README.